rules/expiring-todo-comments.js

```diff
@@ -50,24 +50,12 @@ const messages = {
 
 /** @param {string} dirname */
 function getPackageHelpers(dirname) {
-	// We don't need to normalize the package.json data, because we are only using 2 properties and those 2 properties
-	// aren't validated by the normalization. But when this plugin is used in a monorepo, the name field in the
-	// package.json can be invalid and would make this plugin throw an error. See also #1871
-	/** @type {readPkgUp.ReadResult | undefined} */
-	let packageResult;
-	try {
-		packageResult = readPackageUpSync({normalize: false, cwd: dirname});
-	} catch {
-		// This can happen if package.json files have comments in them etc.
-		packageResult = undefined;
-	}
-
-	const hasPackage = Boolean(packageResult);
-	const packageJson = packageResult ? packageResult.packageJson : {};
+	const packageJsonResult = readPackageJson(dirname);
+	const hasPackage = Boolean(packageJsonResult);
 
 	const packageDependencies = {
-		...packageJson.dependencies,
-		...packageJson.devDependencies,
+		...packageJsonResult.packageJson.dependencies,
+		...packageJsonResult.packageJson.devDependencies,
```

Comment:
Will this line throw if there is no `package.json`?

Reply:
No, it falls back to an empty object.

Reply:
You're right, I was confused by [this line](https://github.com/sindresorhus/eslint-plugin-unicorn/pull/2619/files#diff-983077e885cbad3d4bbd1379551c5912e39e4e0685457d3a0a9f8e320e53ac30R180) in my early morning haze

Reply:
you can test the fix by running these lines in the node repl:

```js
const packageJsonResult = {}
const packageJson = packageJsonResult?.packageJson ?? {}
const test = { ...packageJson.dependencies, ...packageJson.devDependencies }
```